contracts/protocol/v1/proxies/P1LiquidatorProxy.sol

```diff
@@ -140,9 +142,15 @@ contract P1LiquidatorProxy is
     {
         I_PerpetualV1 perpetual = I_PerpetualV1(_PERPETUAL_V1_);
 
+        // Verify that this account can liquidate for the liquidator.
+        require(
+            liquidator == msg.sender || perpetual.hasAccountPermissions(liquidator, msg.sender),
```

Comment:
`liquidator == msg.sender` redundant since that will be checked in `hasAccountPermissions`

Reply:
I agree, but this does prevent it from making an extra call (costing gas) in the case that they are equal. What do you think?

Reply:
Yeah good point 👍